### DISCUSSION TITLE
How to interpret DEP5 report.

### DISCUSSION THREAD

**Discussion by strad-TaehunKim** [fossology/fossology#2632] [Q&A]
Hi. I'm a newbie here.
I'm using fossology to scan my software.
I check below options when uploading software compressed tar.gz file.
![image](https://github.com/fossology/fossology/assets/96408307/3275d7f4-6a07-4579-a524-47b826bf2cac)
After software scanning jobs are done, I download the DEP5 report, but don't know how to interpret it.
Is there any guideline how to interpret DEP5? Any help would be appreciated. Thank you.

**Comment by GMishx** [comment]
It's a report format by Debian. You can find the relevant links in the report header. But you can read more about the report format and specification here: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
